include/abb_librws/rws_subscription.h

```diff
@@ -111,6 +112,13 @@ namespace abb :: rws
      */
     virtual std::string getResourceURI(OperationModeResource const&) const = 0;
 
+    /**
+     * \brief Get URI for subscribing to elog
+     *
+     * \return Subscription URI
+     */
+    virtual std::string getResourceURI(ElogResource const&) const = 0;
+
```

Comment:
You moved these `getResourceURI()` to resource classes in a different PR, no?

Reply:
Yes, @krzyfre moved them, I must merge this code with his changes.